FindTown/FindTownUI/Sources/FindTownUI/UIColor+Extension.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  File.swift
+//  
+//
+//  Created by 장선영 on 2022/12/16.
+//
+
+import Foundation
+import UIKit
+
+public extension UIColor {
+    
+    static let back1: UIColor = .init(named: "Back1", in: .module, compatibleWith: nil) ?? .clear
```

Comment:
```suggestion
    static let back1 = UIColor(named: "Back") ?? .clear
```
`init` 메서드를 안쓰면 조금 더 코드가 간결해질 것 같아요!

Reply:
앗 확실히 그렇네요 !! 수정하겠습니다 !!